source/stylesheets/refills/_modal.scss

```diff
@@ -149,12 +149,12 @@
     }
   }
 
-  .modal-state:checked + .modal-window {
+  .modal-state:checked + .modal-fade-screen {
     opacity: 1;
     visibility: visible;
   }
 
-  .modal-state:checked + .modal-window .modal-inner {
+  .modal-state:checked + .modal-fade-screen .modal-inner {
```

Comment:
Merge rule `.modal-state:checked + .modal-fade-screen .modal-inner` with rule on line 152<br>Selector should have depth of applicability no greater than 2, but was 3